recipes/tensorflow/samples/tensorflow/retail/Dockerfile

```diff
@@ -15,7 +15,7 @@
 # from Adobe.
 #####################################################################
 
-FROM adobe/acp-dsw-ml-runtime-python:0.27.0
+FROM adobe/acp-dsw-ml-runtime-python:0.31.6
```

Comment:
Once the platform sdk python has PALM enabled version, please update this version accordingly.